web/packages/extension/manifest_other.json5

```diff
@@ -50,11 +55,20 @@
     "permissions": [
         "storage",
         "scripting",
+        "declarativeNetRequest",
+        "declarativeNetRequestFeedback",
```

Comment:
Is this permission needed in the release manifest? https://developer.chrome.com/docs/extensions/reference/api/declarativeNetRequest#permissions says it's used for debugging features.

Reply:
Thanks, I have removed it correspondingly.